js/id/ui/alert.js

```diff
@@ -44,10 +44,8 @@ iD.ui.Alert = function(message,type,stack) {
 
     var prettyString = function(obj) {
         var finalString = '';
+        obj.commandDetail =  JSON.stringify(obj.commandDetail, null, '<br>');
```

Comment:
I meant just to revert the recommended change, so both for loop and if stmt like it was before.